### PR TITLE
desktop: Persist volume to disk & don't translate GPU backends

### PR DESCRIPTION
**Pull request by Dinnerbone**
I kept the `--volume` CLI argument though I'm not entirely convinced of its usefulness now. Regardless, if you pass in `--volume` then we don't persist any changes and consider it a temporary thing.

### CODE REVIEW

**Review by danielhjacobs — COMMENTED** [review]
Should this string be removed?

File: desktop/assets/texts/en-US/preferences_dialog.ftl [hunk 1]
```diff
@@ -3,11 +3,6 @@ preferences-dialog = Ruffle Preferences
 preference-locked-by-cli = Read-Only (Set by CLI)
 
 graphics-backend = Graphics Backend
-graphics-backend-vulkan = Vulkan
-graphics-backend-gl = OpenGL
-graphics-backend-dx12 = DirectX 12
-graphics-backend-metal = Metal
-graphics-backend-default = Default
```
Comment (danielhjacobs) [inline desktop/assets/texts/en-US/preferences_dialog.ftl:10]: Should this string be removed?

Comment (Dinnerbone) [reply desktop/assets/texts/en-US/preferences_dialog.ftl:10]: Hmm good point!

Comment (Dinnerbone) [reply desktop/assets/texts/en-US/preferences_dialog.ftl:10]: Added it back!

**Pull request merged**